zello-sdk/src/com/zello/sdk/AppState.java

```diff
@@ -319,7 +322,7 @@ public String getUsername() {
 	 * </p>
 	 * @return <code>Error</code> type indicating the latest error.
      */
-	public Error getLastError() {
+	public @NonNull Error getLastError() {
```

Comment:
Maybe not important, but I find it useful to annotate the member variables themselves as well when annotating the getters/setters. Doesn't have much meaning for `Nullable` getters, but if your getter is marked `NonNull` it helps to ensure that anyone editing the underlying variable will be warned if they try to break that promise.

Reply:
Agreed, I will go through some of those to straighten them out.